docs/tutorials/dicom.ipynb

```diff
@@ -0,0 +1,298 @@
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "qFdPvlXBOdUN"
+      },
+      "source": [
+        "# Gradient Decode Dicom Tensorflow Operation Example"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MfBg1C5NB3X0"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/io/tutorials/dicom\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/io/blob/master/docs/tutorials/dicom.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/io/blob/master/docs/tutorials/dicom.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "      <td>\n",
+        "    <a href=\"https://raw.githubusercontent.com/tensorflow/io/master/docs/tutorials/dicom.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "xHxb-dlhMIzW"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/tree/master/tensorflow_io/dicom) in TensorFlow IO to decode DICOM files with TensorFlow."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup and Usage"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4YsfgDMZW5g6",
+        "colab_type": "text"
+      },
+      "source": [
+        "#### Download DICOM image\n",
+        "\n",
+        "Note: The `CR-MONO1-10-chest` file used in this toturial is downloaded from: https://barre.dev/medical/samples/"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Tu01THzWcE-J",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!curl -o CR-MONO1-10-chest.gz -L https://www.dropbox.com/s/yw9551g5cqaxdn2/CR-MONO1-10-chest.gz?dl=1\n",
+        "!gunzip CR-MONO1-10-chest.gz\n",
+        "!ls -l CR-MONO1-10-chest"
+      ],
```

Comment:
I'd rather not have this downloading from some random dropbox link. 

Would it be possible to add the example file to the github repository and download that?

Also,  is it possible to find an example file with clear licensing?

Reply:
Thanks @MarkDaoust, I have changed the PR to use another file from `NIH Chest X-ray dataset`, provided in https://cloud.google.com/healthcare/docs/resources/public-datasets/nih-chest

As was mentioned in the page, there is a attribution to the citation requirement, so I also added the citation and the link.

The file is stored in Google Cloud's public dataset in gcs. I saved a copy in this repo, conforms to the license requirement I think.